app/src/main/assets/licenses.xml

```diff
@@ -91,15 +91,15 @@
         license="Apache 2.0"
         licenseText="LICENSE_APACHE-2.0.txt" />
     <library
-        name="RxAndroid"
+        name="RxJava"
         author="ReactiveX"
-        website="https://github.com/ReactiveX/RxAndroid"
+        website="https://github.com/ReactiveX/RxJava"
         license="Apache 2.0"
         licenseText="LICENSE_APACHE-2.0.txt" />
     <library
-        name="StackBlur"
```

Comment:
StackBlur is still used: https://github.com/AntennaPod/AntennaPod/blob/6e5004be22f20e3a51697909bf4d1e423252b83c/core/src/main/java/de/danoeh/antennapod/core/glide/FastBlurTransformation.java#L58-L65